app/views/layouts/_header.html.erb

```diff
@@ -143,40 +143,44 @@
         <li class="hidden-xs hidden-sm hidden-md"><a href="//store.publiclab.org"><%= t('layout._header.store') %></a></li>
 
       </ul>
- 
+
       <div id="chat" class="chat-popover" data-placement="bottom" style="display:none;">
         <a style="margin-right:4px;" class="pull-right btn btn-sm" href="https://webchat.oftc.net/?channels=publiclab<% if current_user %>&nick=<%= current_user.username %><% end %>" target="_blank"><%= t('layout._header.chatroom.pop_out') %></a>
         <p class="popover-title"><%= t('layout._header.chatroom.chatroom_title') %></p>
-        
+
       </div>
- 
+
       <% unless current_user %><a rel="tooltip" title="<%= t('layout._header.become_part_community') %>" data-placement="bottom" style="margin-left:14px;" href="/signup" class="btn btn-primary navbar-btn navbar-right hidden-xs"><%= t('layout._header.join') %></a><% end %>
- 
+
       <% unless params[:action] == "register" || params[:action] == "signup" %>
         <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
             <a onClick="$('#chat').toggle();$('#chat').append('<iframe width=\'400px\' height=\'300px\' style=\'border:0;\' src=\'https://webchat.oftc.net/?channels=publiclab<% if current_user %>&nick=<%= current_user.username %><% end %>\'></iframe>');">
               <i class="fa fa-white fa fa-comments"></i>
-            </a> 
+            </a>
           </li>
           <li class="dropdown">
             <% if current_user %>
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">
               <i class="fa fa-user fa fa-white"></i>
                <b class="caret"></b>
-            </a> 
+            </a>
             <ul class="dropdown-menu">
               <li class="dropdown-header"><%= current_user.username %><% if current_user.role === 'moderator' || current_user.role === 'admin'%>: <%= current_user.role %><% end %></li>
               <li><a href="/dashboard"><%= t('layout._header.dashboard') %></a></li>
               <li><a href="/profile/<%= current_user.username %>"><%= t('layout._header.profile') %></a></li>
-              <li><a href="/profile/edit"><%= t('layout._header.edit_profile') %></a></li> 
+              <li><a href="/profile/edit"><%= t('layout._header.edit_profile') %></a></li>
               <li class="divider"></li>
               <li><a href="/subscriptions"><%= t('layout._header.subscriptions') %></a></li>
-              <li><a href="/profile/<%= current_user.username %>/likes"><%= t('layout._header.notes_liked') %></a></li> 
+              <li><a href="/profile/<%= current_user.username %>/likes"><%= t('layout._header.notes_liked') %></a></li>
               <li class="divider"></li>
 	       <% if current_user.role == 'admin' %>
-	        <li class="divider"></li>
-		<li><a href="/useremail">Email search</a></li> 
+		       <li><a href="/useremail">Email search</a></li>
+           <li class="divider"></li>
```

Comment:
Hi, this looks great, but do you think you could put the new links above the divider, and also make the indentations consistent? It really helps us keep the code tidy. 

Thanks a lot -- otherwise this looks excellent!